vcr/pe/presentation_submission.go

```diff
@@ -260,29 +260,15 @@ func resolveCredential(path []string, mapping InputDescriptorMappingObject, valu
 // The credentials will be returned as map with the InputDescriptor.Id as key.
 // The Presentation Definitions are passed in the envelope, as specified by the PEX specification.
 // It assumes credentials of the presentations only map in 1 way to the input descriptors.
-func (s PresentationSubmission) Validate(envelope interface{}, definition PresentationDefinition) (map[string]vc.VerifiableCredential, error) {
-	actualCredentials, err := s.Resolve(envelope)
+func (s PresentationSubmission) Validate(envelope Envelope, definition PresentationDefinition) (map[string]vc.VerifiableCredential, error) {
+	actualCredentials, err := s.Resolve(envelope.Interface)
```

Comment:
Can't the Resolve also work on the Envelope? The switch at the beginning might be removed.